controllers/hardwareclassificationcontroller_controller.go

```diff
@@ -36,6 +36,10 @@ type HardwareClassificationControllerReconciler struct {
 // +kubebuilder:rbac:groups=metal3.io.sigs.k8s.io,resources=hardwareclassificationcontrollers,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=metal3.io.sigs.k8s.io,resources=hardwareclassificationcontrollers/status,verbs=get;update;patch
```

Comment:
Sorry, I just noticed this. What is this group ? It should be just metal3.io . It needs to be modified here and in here : https://github.com/metal3-io/hardware-classification-controller/blob/master/api/v1alpha1/groupversion_info.go#L28  That should be addressed either here or in #7 In any case you have a conflict between the two, so please decide which should go first, let's focus on that one, and then rebase the other one on the first merged.

Reply:
@sharajat  address these changes in this PR only.

Reply:
As #7  PR has lot of changes related to group name, I think it is good to address there because we will have to change CR YAML file name as well as there are params attached to it.